public/index.html

```diff
@@ -7,6 +7,56 @@
   <!-- bootstrap site shows src tags like these at the end of body -->
   <title>Castle of the Winds</title>
 
+<<<<<<< HEAD
```

Comment:
this indicates a merge conflict..... You have to handle those first before committing.